src/Proto3/Wire/Builder.hs

```diff
@@ -385,10 +390,31 @@ word64LE w = Builder (Sum 8, BB.word64LE w)
 --
 -- This encoding is used in the wire format of Protocol Buffers version 3.
 word64Base128LEVar :: Word64 -> Builder
-word64Base128LEVar i
-    | i <= 0x7f = word8 (fromIntegral i)
-    | otherwise = word8 (fromIntegral (0x80 .|. i)) <>
+
+{-
+word64Base128LEVar !i
+    | i < 0x80 = word8 (fromIntegral i)
+    | otherwise = word8 (fromIntegral i .|. 0x80) <>
           word64Base128LEVar (i `shiftR` 7)
+-}
```

Comment:
Optional but recommended: I think this comment can be deleted; we have source control if anyone wants to go back to the old version.

Reply:
yeah its a stray

Reply:
Thanks for the cleanup.